### DISCUSSION TITLE
Include Groupware (EDIT: you can enable it in nextcloud itself)

### DISCUSSION THREAD

**Discussion by thibaultmol** [nextcloud/all-in-one#453] [Ideas] [answered]
(I didn't see it in the readme, so i'm assuming the current project doesn't include it. But it would be very useful if it did)

**Comment by szaimen** [comment]
Hi, not sure what you mean with groupware?

**Comment by thibaultmol** [comment]
https://help.nextcloud.com/t/nextcloud-groupware/99191/3
Guess it's just included (enable-able) then. Haven't yet installed it, so didn't know

**Comment by szaimen** [comment, marked as answer]
Yes, it is included. You can also simply enable or disable apps using the built-in apps store.
